### PR TITLE
Added message input character limit

### PR DESCRIPTION
**Pull request by dsconsole283**
Limit is set to 1500 characters

### CODE REVIEW

File: WPFClient/Pages/ChatPage.xaml.cs [hunk 1]
```diff
@@ -94,20 +80,21 @@ public void DisplayUsers()
         {
             App._connection.On("UpdateUsersList", (List<string> userNames) =>
             {
-                foreach(var localUser in Users){
-                    if(!userNames.Contains(localUser))
+                foreach (var localUser in Users)
```
Comment (nickwhitley) [inline WPFClient/Pages/ChatPage.xaml.cs:83]: These variables, i.e. localUser and remoteUser, should be renamed to be more clear on what they are.

File: WPFClient/Pages/Login.xaml.cs [hunk 2]
```diff
@@ -28,33 +16,47 @@ public Login()
 
         private void loginButton_Click(object sender, RoutedEventArgs e)
         {
-            App._connection.InvokeCoreAsync("ReceiveUsername",
-                                            args: new[] { userNameTextBox.Text });
-            //App._connection.On("ReceiveChatMessage", (string message) =>
-            //{
-            //    WriteLine($"{ message }");
-            //});
+            if (userNameTextBox.Text != "")
```
Comment (nickwhitley) [inline WPFClient/Pages/Login.xaml.cs:19]: Is there a better way to handle this? maybe (!string.IsNullOrEmpty(userNameTextBox.Text))?

**Review by nickwhitley — APPROVED** [review]
Left two comments on files. Also "userNameTextBox" needs to be renamed to "usernameTextBox"

File: WPFClient/Pages/ChatPage.xaml.cs (hunk 1 above)
Comment (dsconsole283) [reply WPFClient/Pages/ChatPage.xaml.cs:83]: Woooooow

File: WPFClient/Pages/Login.xaml.cs (hunk 2 above)
Comment (dsconsole283) [reply WPFClient/Pages/Login.xaml.cs:19]: When left blank, the text property for the text box has a value of " "

**Pull request merged**